specs/nights_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../lib/nights'
+require_relative 'spec_helper'
+
+describe "NIGHTS" do
+  before do
+    @summer_somewhere_nights = Nights.new("January 4, 2018", "January 11, 2018")
+  end
+
+  describe "#INITIALIZE" do
```

Comment:
It was a good call to make a `Nights` class which would deal with the logic of thinking about a date range. I would've liked to see more test coverage around the logic that belongs to this (aka valid dates)